Storage/ThirdParty/SwiftData.swift

```diff
@@ -202,6 +219,14 @@ open class SwiftData {
         }
     }
 
+    public func cancel() {
+        sharedConnectionQueue.async {
```

Comment:
@rnewman why were you suggesting I put this on the DispatchQueue? I don't want serial queue behavior. This line should be removed methinks.

Reply:
So long as the connection (`c.sqliteDB`) is not closed when `sqlite3_interrupt` is called, it's safe to call at any time. Indeed, it simply doesn't make sense for this to be on the queue — once this is running on the queue, no other query is!